src/envo/envo-edit.owl

```diff
@@ -39732,6 +39734,29 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_04000018> <http://purl.obolibrar
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_04000018> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000066> <http://purl.obolibrary.org/obo/ENVO_00000015>))
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_04000018> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000057> <http://purl.obolibrary.org/obo/ENVO_01001497>))
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105002> (thermoplasticity)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://www.lexico.com/en/definition/thermoplastic>) Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://www.merriam-webster.com/dictionary/thermoplastic>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105002> "A disposition inhering in a bearer by virtue of the bearer's ability to, repeatedly, become plastic on heating and harden again on cooling."@en)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.obolibrary.org/obo/ENVO_06105002> <https://orcid.org/0000-0002-2996-719X>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.obolibrary.org/obo/ENVO_06105002> <https://orcid.org/0000-0003-1072-8081>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_06105002> <http://orcid.org/0000-0002-8166-0486>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_06105002> "2021-04-27T09:10:14Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_06105002> "熱塑性"@zh)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/ENVO_06105002> "envoPlastics"@en)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105002> "thermoplasticity"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105002> <http://purl.obolibrary.org/obo/BFO_0000016>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105003> (thermoplastic material)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://en.wikipedia.org/wiki/Thermoplastic>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105003> "A material which have thermoplastic disposition."@en)
```

Comment:
Please follow the definition form more closely. As you are using `environmental material` as the parent term, the definition should read as "An environmental material which ..." (following "A `B`which `Cs`").

Even though you have already defined `thermoplasticity`, I would expand on that again here. So instead of keeping it as short as it currently is, saying "... is capable of softening or fusing when heated and of hardening again when cooled".

Source: https://www.merriam-webster.com/dictionary/thermoplastic
> capable of softening or fusing when heated and of hardening again when cooled

Reply:
Is the above resolved?

Reply:
@pbuttigieg Yes the comment is resolved. Please let me know if you have any other comments or suggestions.